RoaringBitmap/src/main/java/org/roaringbitmap/buffer/MappeableArrayContainer.java

```diff
@@ -521,9 +513,15 @@ public MappeableContainer flip(char x) {
         content.put(-loc - 1, x);
         ++cardinality;
       } else {
-        for (int k = loc + 1; k < cardinality; --k) {
-          content.put(k - 1, content.get(k));
+        // buffer is read only, we must make mutable copy
```

Comment:
This is not relevant to this PR. Please add it to a separate PR.

Further, if you think that it is necessary, please demonstrate using our main API.